types/htmlhint/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for HTMLHint 1.1
+// Project: https://github.com/yaniswang/HTMLHint
+// Definitions by: Alan Agius <https://github.com/alan-agius4/>
```

Comment:
you would have to ping him explicitely here to get ACK to be llisted as owner (notifications and that kind of stuff)

Reply:
https://github.com/microsoft/vscode-htmlhint/issues/2